programs/server/config.xml

```diff
@@ -70,12 +70,15 @@
         You can specify log format(for now, JSON only). In that case, the console log will be printed
         in specified format like JSON.
         For example, as below:
+
         {"date_time":"1650918987.180175","thread_name":"#1","thread_id":"254545","level":"Trace","query_id":"","logger_name":"BaseDaemon","message":"Received signal 2","source_file":"../base/daemon/BaseDaemon.cpp; virtual void SignalListener::run()","source_line":"192"}
+        {"date_time_utc":"2022-04-26T20:36:27+00:00","thread_name":"#1","thread_id":"254545","level":"Trace","query_id":"","logger_name":"BaseDaemon","message":"Received signal 2","source_file":"../base/daemon/BaseDaemon.cpp; virtual void SignalListener::run()","source_line":"192"}
```

Comment:
What is the actual format that will be produced? The PR's body suggests 
```json
{"date_time_utc":"2024-10-31T15:20:50Z.281688", 
```
But here we have 
```json
{"date_time_utc":"2022-04-26T20:36:27+00:00",
```

Reply:
Ah, looks like the implementation points at the former `"2024-10-31T15:20:50Z.281688"`